app/grandchallenge/cases/models.py

```diff
@@ -287,6 +287,8 @@ class Image(UUIDModel):
     color_space = models.CharField(
         max_length=5, blank=False, choices=COLOR_SPACES
     )
+    min_voxel_value = models.IntegerField(null=True, blank=True)
+    max_voxel_value = models.IntegerField(null=True, blank=True)
```

Comment:
These can be positive and negative - is that intended or should this be a `PositiveIntegerField`?

Reply:
It's intended. They can be negative when working with signed integer datatypes.